inst/templates/bioc-support.md

```diff
@@ -21,15 +21,24 @@ Armed with your reprex, the next step is to figure out [where to ask](https://ww
 *   If you're not sure: let the community help you figure it out!
     If your problem _is_ a bug or a feature request, you can easily return here and report it.
 
-Before opening a new issue, be sure to [search issues and pull requests](https://github.com/{{github_spec}}/issues) to make sure the bug hasn't been reported and/or already fixed in the development version.
+Before opening a new issue, be sure to [search issues and pull
+requests](https://github.com/{{github_spec}}/issues) to make sure the bug
+hasn't been reported and/or already fixed in the development version.
 By default, the search will be pre-populated with `is:issue is:open`.
-You can [edit the qualifiers](https://help.github.com/articles/searching-issues-and-pull-requests/)  (e.g. `is:pr`, `is:closed`) as needed.
-For example, you'd simply remove `is:open` to search _all_ issues in the repo, open or closed.
+You can [edit the
+qualifiers](https://help.github.com/articles/searching-issues-and-pull-requests/)
+(e.g. `is:pr`, `is:closed`) as needed.
+For example, you'd simply remove `is:open` to search _all_ issues in the repo,
+open or closed.
 
 ## What happens next?
```

Comment:
This section is all about tidyverse, which is irrelevant when asking about Bioconductor packages...